gapic/src/main/com/google/gapid/perfetto/views/RootPanel.java

```diff
@@ -106,6 +119,36 @@ public void setSize(double w, double h) {
     state.setMaxScrollOffset(bottomHeight - h + topHeight);
   }
 
+  private SortedMap<Long, Boolean> searchForFlag(double x) {
+    long time = state.pxToTime(x);
+    // If the click falls within the flag icon's boundaries, the flag is considered to be hit.
+    // Leave an additional two pixels to the left of the flag to get a better hit box
+    long rightOffset = state.deltaPxToDuration(FLAG_WIDTH);
+    long leftOffset = state.deltaPxToDuration(2);
```

Comment:
optional: Might want to increase this a bit to 5 or 6.  It feels a little odd that there is such an imbalance between how close you can set a flag on the left vs on the right.  Ignore me if you don't feel the same.

Reply:
The imbalance is because the 'x' is the pole of the flag. The flag is facing right, so the the right offset is a bit more. The left offset of 2 pixels is just to make sure that clicking on the exact pole of the flag is considered as well.